test/index.js

```diff
@@ -1,5 +1,18 @@
 'use strict';
-const binding = require('./build/Release/binding.node');
-const assert = require('assert');
+global.buildType = process.config.target_defaults.default_configuration;
+global.binding = require(`./build/${buildType}/binding.node`);
+global.assert = require('assert');
```

Comment:
Yes, I didn't really like this use of global either. Ability to run the test files individually is a good point.